config/settings.yml

```diff
@@ -11,3 +11,5 @@ gemfile_checker:
   pr_contacts: []
 issue_manager:
   repo_names: []
+label_notifier:
+  contacts: { label1: [] }
```

Comment:
Please just leave this as an empty value.  `contacts: {}`

The settings are actually used, so the bot will just start looking for a label called "label1"